clueless/state.py

```diff
@@ -41,33 +51,63 @@ def sock_addr(self) -> str:
 
     @property
     def is_registered(self) -> bool:
-        return self.character is not None
+        return self.state is UserState.REGISTERED
 
     async def dispatch_message(self, message):
+        print("Dispatch_message function:")
```

Comment:
`self.logger` should be used for debug messages. There are different levels that can be logged to with `self.logger.info`, `self.logger.debug`, etc. Docs on it are [here](https://docs.python.org/3.9/library/logging.html)